pavement.py

```diff
@@ -3,41 +3,49 @@
 import os
 import platform
 import fnmatch
+import shutil
 import zipfile
 
 from paver.easy import *
 
-
-def get_extlibs():
-    if platform.system() == "Windows":
-        return 'extlibs_windows'
-    if platform.system() == "Darwin":
-        return 'extlibs_macos'
-    if platform.system() == "Linux":
-        return 'extlibs_linux'
-
-
 options(
     plugin=Bunch(
         name='ee_plugin',
-        ext_libs=path(get_extlibs()),
+        ext_libs=path('extlibs'),
         source_dir=path('.'),
         package_dir=path('.'),
         tests=['test', 'tests'],
         excludes=[
             "*.pyc",
             ".git",
+            ".github",
             ".idea",
             ".gitignore",
             ".travis.yml",
             "__pycache__",
+            "docs",
+            "help",
+            "test",
             "media",
             "ee_plugin.zip"
         ]
     ),
 )
 
 
+def clean_extlibs():
```

Comment:
Can we do without this? I find it a bit scary to do an `os.walk` + `os.remove` on users computers. 
Or perhaps remove the specific directory?

Reply:
After some testing, I realized that the dependencies of this plugin are not OS-depend, you don't want to have windows DLLs in your MacOS plugin dir and so on. So this deletes any unneeded linked lib and caches. Beside:

- This function runs only in the ext_lib directory created when pave is call
- This function is part of the plugin packaging tool that runs in a docker container inside Github action, in theory a user never needs to run this
- Another purpose of this is to reduce the plugin size